src/smt/smt_engine.cpp

```diff
@@ -4382,22 +4362,15 @@ void SmtEnginePrivate::processAssertions() {
   }
   Debug("smt") << " d_assertions     : " << d_assertions.size() << endl;
 
-  Trace("smt-proc") << "SmtEnginePrivate::processAssertions() : pre-ite-removal" << endl;
-  dumpAssertions("pre-ite-removal", d_assertions);
   {
-    Chat() << "removing term ITEs..." << endl;
-    TimerStat::CodeTimer codeTimer(d_smt.d_stats->d_iteRemovalTime);
```

Comment:
Why are you removing the time stats?

Reply:
Because the PreprocessingPass class automatically provides a timer for each pass, no need to do it explicitly.

Reply:
Ah, that's good to know!